client/src/components/Header/Header.stories.js

```diff
@@ -3,16 +3,8 @@ import React from "react";
 import Header from "./Header";
 
 export default {
-  title: "Example/Header",
+  title: "Example/Components/Header",
```

Comment:
It depends on your structure in Storybook, but I guess these are not example stories anymore.

```suggestion
  title: "Components/Header",
```